### DISCUSSION TITLE
Nodemailer from email name and address

### DISCUSSION THREAD

**Discussion by MichaelFBA** [directus/directus#16941] [Feature Requests]
At the moment the the from field in all emails is [hard coded](https://github.com/directus/directus/blob/main/api/src/services/mail/index.ts#L55) to use the directus project name. In order to support multi tenancy emails it would be great to support the native [address](https://nodemailer.com/message/addresses/) format in the from field.

Address format 
```
{
    name: 'Some name',
    address: 'foobar@example.com'
}
```
This is in response to this PR that was rejected because it didnt have an associated discussion
https://github.com/directus/directus/pull/16863#event-8105351019

**Comment by filiphanes** [comment]
I would like to add From field to Send email flow operation, too.
It would use default from when empty.

**Comment by rijkvanzanten** [comment]
Heya! Thank you for taking the time to submit this request! 

It has been over **90 days**, and this discussion has not received at least **15 votes** from the community. This means that we don't feel like there's enough community interest to warrant further R&D into this topic at this time. :ice_cube:

This request will now be closed to keep our discussions tidy. Please reach out if you have any questions!

_For more information, see our [Feature Request Process](https://docs.directus.io/contributing/feature-request-process.html)._
